www/src/app/services/alfred-api-service.js

```diff
@@ -0,0 +1,30 @@
+import { API_URL } from '../../config';
+import { get, post } from '../utils';
+
+
+export function fetchUpdatesByAuthor(author) {
+  const updatesByAuthor = `${API_URL}/updates?author=${author}`;
+  const currentReportsInfo = `${API_URL}/reports/current`;
+
+  return get(currentReportsInfo).then(([prevReport, nextReport]) => {
```

Comment:
Should we move the action of getting the current reports info into its own service function, and possible its own Redux action which is dispatched when `InboxContainer` is mounted or when the inbox route is entered?